plugins/plugin-java/che-plugin-java-ext-lang-client/src/main/java/org/eclipse/che/ide/ext/java/client/refactoring/rename/JavaRefactoringRename.java

```diff
@@ -307,10 +309,13 @@ private void performRename(String newName) {
     extensionServiceClient
         .rename(settings)
         .then(
-            edits -> {
+            result -> {
               enableAutoSave();
               undoChanges();
-              applyWorkspaceEditAction.applyWorkspaceEdit(edits);
+              RefactoringStatus refactoringStatus = result.getRefactoringStatus();
+              if (!FATAL.equals(refactoringStatus.getRefactoringSeverity())) {
```

Comment:
How do we inform the user that something went wrong in this case?

Reply:
added a way to show notification if error message